src/oca_github_bot/config.py

```diff
@@ -47,12 +47,7 @@ def func_wrapper(*args, **kwargs):
 
 DRY_RUN = os.environ.get("DRY_RUN", "").lower() in ("1", "true", "yes")
 
-# Coma separated list of task to run
-# By default all configured tasks are run.
-# Available tasks:
-#  delete_branch,tag_approved,tag_ready_to_merge,gen_addons_table,
-#  gen_addons_readme,gen_addons_icon,setuptools_odoo,merge_bot,tag_needs_review
-BOT_TASKS = os.environ.get("BOT_TASKS", "all").split(",")
+BOT_TASKS_IGNORED = os.environ.get("BOT_TASKS_IGNORED", "").split(",")
```

Comment:
I see why providing tasks to ignore can be useful. However explicitly enumerating tasks to activate is useful too, to avoid having new tasks activated by default during an upgrade. I guess we can have both, with BOT_TASK_IGNORED coming after BOT_TASKS in the config loading and deactivating some. BOT_TASK_DISABLED might be a better name BTW.

Reply:
OK.  Just a question, did you misspelled, or do you really want ``BOT_TASK_DISABLED`` without ``S``. For the time being, the other variable is plural ``BOT_TASKS``